cmd/neofs-node/config.go

```diff
@@ -931,13 +933,21 @@ func writeSystemAttributes(c *cfg) error {
 	var paths []string
 	for _, sh := range c.applicationConfiguration.EngineCfg.shards {
 		for _, subStorage := range sh.SubStorages {
-			path := subStorage.Path
+			var path = subStorage.Path
```

Comment:
why this change?

Reply:
Because it's better this way! I've minimized changes here now.